homeassistant/components/roborock/config_flow.py

```diff
@@ -153,3 +166,108 @@ def _create_entry(
                 CONF_BASE_URL: client.base_url,
             },
         )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Create the options flow."""
+        return RoborockOptionsFlowHandler(config_entry)
+
+
+class RoborockOptionsFlowHandler(config_entries.OptionsFlow):
```

Comment:
There's an OptionsFlowWithConfigEntry which does exactly what you do here (so you can remove the constructor)

Reply:
Good to know - thanks!